py4cast/datasets/base.py

```diff
@@ -808,6 +807,86 @@ def projection(self):
         return func(**self.projection_kwargs)
 
 
+@dataclass(slots=True)
+class Settings:
+    dataset_name: str
+    num_input_steps: int  # Number of input timesteps
+    num_pred_steps: int  # Number of output timesteps
+    step_duration: float  # duration in hour
+    standardize: bool = True
+    file_format: Literal["npy", "grib"] = "grib"
+    members: Tuple[int] = (0,)
+
+
+ParamConfig = namedtuple(
```

Comment:
Same, adding a comment to explain what it is could be useful ?

Reply:
Indeed